### PR TITLE
tair fix distance_type error, and add hybrid search

### PR DESCRIPTION
**Pull request by tianhanwen**
- fix: distance_type error, 
- feature: Tair add hybrid search

### CODE REVIEW

File: libs/langchain/langchain/vectorstores/tair.py [hunk 1]
```diff
@@ -86,23 +86,38 @@ def add_texts(
         """Add texts data to an existing index."""
         ids = []
         keys = kwargs.get("keys", None)
+        use_hybrid_search = kwargs.get("use_hybrid_search", False)
```
Comment (baskaryan) [inline libs/langchain/langchain/vectorstores/tair.py:89]: is this something that would make more sense to make an instance attribute of the object? would you ever want to add some texts with hybrid_search and some without in the same vector db?

Comment (tianhanwen) [reply libs/langchain/langchain/vectorstores/tair.py]: 1. Updated the code logic of add_texts to obtain the lexical_algorithm passed by the user, and no longer use the use_hybrid_search parameter
2. The VECTOR attribute (must be capitalized) is an internal reserved attribute used to store vectors. TEXT attribute (must be uppercase), internal reserved attribute, using bm25 algorithm needs to store the original text. Both VECTOR and TEXT can be stored in one index

Comment (tianhanwen) [reply libs/langchain/langchain/vectorstores/tair.py]: Now, inside the add_texts method, use tvs_get_index to get the lexical_algorithm parameter

**Comment by baskaryan** [comment]
thanks @tianhanwen!

**Pull request merged**